H/HiGHS/build_tarballs.jl

```diff
@@ -4,12 +4,12 @@ using BinaryBuilder, Pkg
 
 name = "HiGHS"
 
-version = v"1.1.0"
+version = v"1.1.1"
 
 sources = [
     GitSource(
         "https://github.com/ERGO-Code/HiGHS.git",
-        "bd9703cc1a896c68473144064011cc9d64246e07",
+        "6c2ab5414fd9670ad961920aefe0c57d082eb6cc",
```

Comment:
@lgottwald I bumped the commit to your PR. The other option we have is to use a newer version of GCC, although it's preferred to build on the oldest possible to maximize compatibility.